Composer/packages/extensions/adaptive-form/src/components/fields/RecognizerField.tsx

```diff
@@ -37,11 +63,25 @@ const RecognizerField: React.FC<FieldProps<MicrosoftIRecognizer>> = (props) => {
       return;
     }
 
+    selected.map((s) => {
+      if (s === SDKKinds.LuisRecognizer) {
+        onChange(`${currentDialog.id}.lu.qna`);
```

Comment:
side effect inside a useMemo hook. is should be merged to your previous useEffect hook